java/app/src/main/java/com/fullstorydev/shoppedemo/ui/checkout/CheckoutViewModel.java

```diff
@@ -0,0 +1,101 @@
+package com.fullstorydev.shoppedemo.ui.checkout;
+
+import android.app.Application;
+
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.LiveData;
+
+import com.fullstorydev.shoppedemo.data.CustomerInfo;
+import com.fullstorydev.shoppedemo.data.CustomerInfoRepository;
+
+public class CheckoutViewModel extends AndroidViewModel {
+    private CustomerInfoRepository mCustomerInfoRepo;
+    private LiveData<Boolean> isLoading;
+    private CustomerInfo customerInfo;
+
+    public CheckoutViewModel(Application application) {
+        super(application);
+        mCustomerInfoRepo = new CustomerInfoRepository(application);
+        isLoading = mCustomerInfoRepo.getIsLoading();
+        customerInfo = new CustomerInfo.OrderBuilder().buildOrder(); // init to new customerInfo Obj with empty values
```

Comment:
Should this empty value init be done in mCustomerInfoRepo.getCustomerInfo()? If no customer data is in the repo, return defaults?